pennylane_lightning/lightning_kokkos/_state_vector.py

```diff
@@ -63,10 +64,11 @@ def __init__(
         self,
         num_wires: int,
         dtype: Union[np.complex128, np.complex64] = np.complex128,
+        rng: Generator = None,
```

Comment:
Should the Kokkos RNG not defer to an (exposed) internal Kokkos implementation? https://kokkos.org/kokkos-core-wiki/API/algorithms/Random-Number.html

That way we get the GPU doing RNG when requested, rather than the CPU

Reply:
Sounds reasonable :+1: